integrationtests/src/test/java/io/kroxylicious/proxy/KrpcFilterIT.java

```diff
@@ -298,6 +304,54 @@ public void shouldModifyFetchMessage() throws Exception {
 
     }
 
+    @Test
+    public void proxySslClusterPlain() throws Exception {
+        String proxyAddress = "localhost:9192";
+
+        var certificateGenerator = new KeytoolCertificateGenerator();
+        certificateGenerator.generateSelfSignedCertificateEntry("test@redhat.com", "localhost", "KI", "RedHat", null, null, "US");
+
+        try (var cluster = KafkaClusterFactory.create(KafkaClusterConfig.builder().testInfo(testInfo).build())) {
+            cluster.start();
+
+            String config = """
+                    proxy:
+                      address: %s
+                      keyStoreFile: %s
+                      keyPassword: %s
+                    clusters:
+                      demo:
+                        bootstrap_servers: %s
+                    filters:
+                    - type: ApiVersions
+                    - type: BrokerAddress
+                    """.formatted(proxyAddress,
+                    certificateGenerator.getCertLocation(), certificateGenerator.getPassword(),
+                    cluster.getBootstrapServers());
+
+            var proxy = startProxy(config);
```

Comment:
Can we use try-with-resources here? If not should we be able to? (I don't see a fix for the latter as in scope for this PR)